tinyms/serving/servable/servable.py

```diff
@@ -17,13 +17,15 @@
 
 import tinyms as ts
 from tinyms import model
+from tinyms.utils.predict.predict import cyclegan_predict
```

Comment:
I don't think it's appropriate to add another `cyclegan_predict` function in `utils` module.